packages/@apphosting/adapter-nextjs/src/overrides.ts

```diff
@@ -0,0 +1,41 @@
+import { AdapterMetadata, MiddlewareManifest } from "./interfaces.js";
+import { loadRouteManifest, writeRouteManifest, loadMiddlewareManifest } from "./utils.js";
+
+/**
+ * Adds the route overrides to the route manifest.
```

Comment:
Can you explain what this function achieves for App Hosting? Its not very clear what adding the route overrides to the route manifest does and why we need to do it.